src/components/pages/blogs/blog/components/ListTemplate.tsx

```
@@ -0,0 +1,37 @@
+    content: ListContent
+}
+const ListTemplate: React.FC<IListTemplate> = ({ content }) => {
+    return (
+        <div className="border-t pt-5">
+            <h2 className="font-black text-[18px] lg:text-[25px] leading-[48px]">
+                {content.title}
+            </h2>
+
+            {content.topDescription && content.topDescription?.split("\n").map((item, index) => (
+                <p className="font-normal text-base leading-8  mb-5" key={index} dangerouslySetInnerHTML={{ __html: item }} />
+            ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)